Tests/BasicsTests/VFSTests.swift

```diff
@@ -70,8 +73,10 @@ class VFSTests: XCTestCase {
             // isExecutableFile
             let executablePath = AbsolutePath("/exec-foo")
             let executableSymPath = AbsolutePath("/exec-sym")
+#if !os(Windows)
             XCTAssertTrue(vfs.isExecutableFile(executablePath))
             XCTAssertTrue(vfs.isExecutableFile(executableSymPath))
+#endif
```

Comment:
this disables both checks, expected?

Reply:
Yes, we aren't generating binaries, so the test is going to fail.  Should the check change, it would suddenly change, so it is safer to just disable the two checks here on Windows I think.

Reply:
doesn't this mean that the feature would be broken on windows though?

Reply:
No, there is no concept of an "executable bit" in Windows.  `chmod +x` is meaningless, it does nothing.  The only indicator of an executable is that it is a COM executable.  This is validating that a file marked with +x is serialized as executable.  We _could_ replace the place holder files with tiny DOS executables though I suppose.  I did do this for swift-driver.

Reply:
gotcha. my meta point is that we need to make sure VFS works correctly on Window too including the `isExecutableFile` API

Reply:
That makes sense.  I can dig out the binary and embed that in the test case.  The other platforms can simply use chmod on it for the test.